### PR TITLE
reset filter when you creat a seek in a filtered out language

### PR DESCRIPTION
**Pull request by jonager**
@domino14 I'm able to reset `lobbyFilterByLexicon` in the state and localStorage when I create a new seek, but active games and sought games components are not updating, they still show the filter. If I change between the taps Play and Watch it updates and goes away. For some reason the change in the store is not triggering a rerender

### CODE REVIEW

**Comment by domino14** [comment]
Sorry for the long delay here.

What happens in this branch?

```javascript
    case ActionType.setLobbyFilterByLexicon: {
```

Do you know if it actually calls that? Have you put a log on it?

File: liwords-ui/src/lobby/active_games.tsx [hunk 1]
```diff
@@ -40,9 +43,9 @@ export const ActiveGames = (props: Props) => {
 
   const handleChange = useCallback(
     (
-      pagination: TablePaginationConfig,
+      _pagination: TablePaginationConfig,
       filters: Record<string, FilterValue | null>,
-      sorter:
+      _sorter:
```
Comment (domino14) [inline liwords-ui/src/lobby/active_games.tsx:48]: would like to understand a little bit better why these are underscored -- `_pagination` and `_sorter`.

**Review by domino14 — COMMENTED** [review]
just have a question

File: liwords-ui/src/lobby/active_games.tsx (hunk 1 above)
Comment (jonager) [reply liwords-ui/src/lobby/active_games.tsx:48]: becuase it's not being used.

**Pull request merged**